dashboard/src/locales/messages/index.ts

```diff
@@ -114,6 +114,8 @@ export const messages = {
     'global.origins': 'Origins',
     'global.pass': 'Pass',
     'global.placeholderSearch': 'Search by tree, branch or tag',
+    'global.projectUnderDevelopment':
+      'This is an ongoing project. Please report bugs and suggestions to bugs@kernelci.org',
```

Comment:
Did anyone suggest this email? If not, I would say to use the same email we are using to receive feedback (`FEEDBACK_EMAIL_TO` in dashboard/src/utils/constants/general.ts)